pkg/cmd/server/apis/config/types.go

```diff
@@ -526,6 +526,10 @@ type ImagePolicyConfig struct {
 	// is exposed externally. The value is used in 'publicDockerImageRepository'
 	// field in ImageStreams. The value must be in "hostname[:port]" format.
 	ExternalRegistryHostname string
+
+	// AdditionalTrustedCAs is a path to a pem file containing additional CAs that
+	// should be trusted during imagestream import.
+	AdditionalTrustedCAs string
```

Comment:
CA singular.  Doc should indicate that is can be a bundle.

Doing it like this means that the trust domain of all CA bundles specified here is uniform.  Any can sign for any names related to image importing.  I'm ok with that.  Just calling it out.

Reply:
updated